### DISCUSSION TITLE
Link that open OsmAND and navigate.

### DISCUSSION THREAD

**Discussion by meyverick** [osmandapp/OsmAnd#17027] [Q&A]
Is there a feature allow us to automatically open OsmAND and navigate to a destination?

I mean somehow like osmand://navigate/[destination] where destination is an address or coordinate?

**Comment by dmpr0** [comment]
OsmAnd can open route links, you can read more here https://docs.osmand.net/docs/user/navigation/setup/route-details#share-link
